src/Paging.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from "react";
+import './App.css';
+
+
+export default class Paging extends Component {
+    constructor() {
+        super();
+        this.state = { page: 1 };
+    }
```

Comment:
You don't need to use this constructor. You can achieve the same thing by just declaring state as an object like this:
```
    state = {
        page: 1
    }
```